apps/charterafrica/src/lib/data/common/processPageContributors.js

```diff
@@ -50,7 +50,7 @@ export async function getContributors(page, api, context) {
         href,
       },
       description: person.description || " ",
-      name: person.fullName ?? null,
+      name: person?.fullName ?? person?.externalId ?? null,
```

Comment:
```suggestion
      name: person.fullName || person.externalId || null,
```

We **_know_** `person` exists so why `person?.`?